animareflection/ui
# `CopyToClipboard` Component

## Description

##### Task link: https://trello.com/c/xe59l6fQ/248-copy-to-clipboard-button

Added `CopyToClipboard` component, stories, and tests. Updated exported hooks (backfill from `usehooks-ts`).

## Test Steps

1) Verify that component and story structure (and logic) is sound.
2) Verify that component renders appropriately in storybook and in downstream example apps.
3) Verify that tests pass.
4) Verify that build includes all hooks from `usehooks-ts`.
